marabunta/parser.py

```diff
@@ -176,8 +176,8 @@ def _parse_version(self, parsed_version, options):
         # parse the modes operations and addons
         modes = parsed_version.get('modes', {})
         if not isinstance(modes, dict):
-            raise ParseError(u"'modes' key must be a dict", YAML_EXAMPLE)
-        for mode_name, mode in modes.items():
+            raise ParseError("'modes' key must be a dict", YAML_EXAMPLE)
+        for mode_name, mode in list(modes.items()):
```

Comment:
Why calling list here ?

maybe we should stick to iteritems or  importing it from future.utils if think efficiency is important (large list)
otherwise sticking with items should be ok.

An other option would be to use `viewitems`

http://python-future.org/compatible_idioms.html#iterating-through-dict-keys-values-items